deploy/helm/flightctl/templates/flightctl-prometheus-config.yaml

```
@@ -0,0 +1,50 @@
+      - job_name: 'flightctl-services'
+        kubernetes_sd_configs:
+          - role: pod
+        relabel_configs:
+          - source_labels: [__meta_kubernetes_pod_container_port_number]
+            action: keep
+            regex: 1569[0-2]
+          - source_labels: [__meta_kubernetes_pod_label_flightctl_service]
+            action: keep
+            regex: .+
+        scheme: http
+{{- end }} 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)